source/concepts/recording.rst

```diff
@@ -69,8 +68,42 @@ In order to send both streams together, provide the two file paths separated by
 **Using the graphical interface**, add a recording to the list of simulations using the ``+ Recording`` button,
 then choose the files.
 
+Recording with the Python client
+--------------------------------
+
+How to record
+~~~~~~~~~~~~~
+
+You can also record NanoVer sessions using a Python client from the nanover.omni module.
+Here is an example of how to define the file names and paths for the recording and pass them to the recording function:
+
+.. code:: python
+
+    from nanover.omni.record import record_from_server
+    # Define the .traj and .state file names and paths
+    traj_path = 'simulation_recording.traj'
+    state_path = 'simulation_recording.state'
+    # Pass the runner to the recording function
```

Comment:
This comment is inaccurate, maybe a typo? Might be better to say the it "creates a client that connects to a server and records until the server stops" or something